### PR TITLE
Implement #73 integration tests for .NET

### PR DESCRIPTION
**Pull request by Mpdreamz**
This adds .NET auto attachment tests

### CODE REVIEW

**Comment by Mpdreamz** [comment]
Finally got around to getting this to work 🎉 @jackshirazi @trentm tagging you to review since you done the `java` and `nodejs` counterpart of this 😸

File: charts/apm-attacher/values.yaml [hunk 1]
```diff
@@ -41,6 +41,6 @@ webhookConfig:
       environment:
         CORECLR_ENABLE_PROFILING: "1"
         CORECLR_PROFILER: "{FA65FE15-F085-4681-9B20-95E04F6C03CC}"
-        CORECLR_PROFILER_PATH: "/usr/agent/apm-dotnet-agent/libelastic_apm_profiler.so"
-        ELASTIC_APM_PROFILER_HOME: "/usr/agent/apm-dotnet-agent"
-        ELASTIC_APM_PROFILER_INTEGRATIONS: "/usr/agent/apm-dotnet-agent/integrations.yml"
+        CORECLR_PROFILER_PATH: "/elastic/apm/agent/apm-dotnet-agent/libelastic_apm_profiler.so"
+        ELASTIC_APM_PROFILER_HOME: "/elastic/apm/agent/apm-dotnet-agent"
+        ELASTIC_APM_PROFILER_INTEGRATIONS: "/elastic/apm/agent/apm-dotnet-agent/integrations.yml"
```
Comment (trentm) [inline charts/apm-attacher/values.yaml:46]: Was the attacher just not working for .NET before this change, then? I.e. will we need/want a new release of the attacher with this change?

**Review by trentm — APPROVED** [review]

File: charts/apm-attacher/values.yaml (hunk 1 above)
Comment (Mpdreamz) [reply charts/apm-attacher/values.yaml:46]: Yeah, I will double check but I am 90÷ sure this never worked.

**Review by jackshirazi — APPROVED** [review]
woohoo!

**Pull request merged**